pages/contribute.md

```diff
@@ -0,0 +1,114 @@
+- Follow the [API Documentation Contribution Guide]({{ site.docs.userguide.url }}/stable/contributing_code.html#working-with-mdanalysis-docs).
+- Browse open API documentation issues in [GitHub Issues]({{ site.github.issues }}).
+
+## Mentoring & Outreach Programs
+MDAnalysis actively participates in various mentoring and outreach programs **to help students, researchers, and early-career contributors gain experience in open-source development and scientific software**. Through these programs, participants receive mentorship, contribute to real-world projects, and develop valuable skills in computational science and software engineering.
+
+### Available Programs
+We have been involved in several mentoring initiatives:
+
+- **[Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/)** \
+    A global, remote program where students contribute to open-source projects under mentorship, enhancing their programming skills while engaging with the open-source community.
+
+- **[Google Season of Docs](https://developers.google.com/season-of-docs)** \
+    A program that connects technical writers with open-source projects to improve documentation quality and accessibility.
+
+- **[Outreachy](https://www.outreachy.org/)** \
+    A diversity-focused program offering paid remote internships to individuals underrepresented in tech, providing opportunities to contribute to open-source software.
+
+- **[Station1 Frontiers Fellowship](https://www.station1.org/sff)**  \
+    A unique research and internship program designed for students in science and technology, promoting inclusive and socially impactful research.
+
+- **[CompChemURG](https://www.bindingsites.co.uk/home)**  \
+    A mentoring initiative focused on computational chemistry, supporting undergraduates and early-career researchers in gaining expertise in the field.
+
+Many of our [core developers][] started through these programs &mdash; consider joining and becoming part of the MDAnalysis community!
+
+### Upcoming Opportunities
+
+Applications for [Google Summer of Code 2025][] are open! If you're interested in participating, check out our project ideas and application guidelines.
+
+For additional updates on upcoming events, visit our [Community]({{ site.baseurl }}/pages/community/) page.
+
+### Past GSoC
+
+MDAnalysis has participated in Google Summer of Code for several years, mentoring students on a variety of open-source projects. Explore our past projects:
+
+- [GSoC 2024][] 
+- [GSoC 2023][] 
+- [GSoC 2022][] 
+
+## Other Contributions
+
+If you are interested in contributing in other ways, such as improving the website or blog, you can refer to the [README file][] in the [Website Repository][]. While the website is already maintained by the MDAnalysis team, minor fixes and updates are always welcome. 
+
+If you'd like to write a **blog post**, feel free to reach out via the [Community]({{ site.baseurl }}/pages/community/) page.
```

Comment:
@jennaswa  a blog post about what? 
I suggest rephrasing to be more explicit. 'documenting your experience of learning MDAnalysis' or 'to tell us about your involvement in a UGM' or similar.